.github/workflows/admin-rollout-k8s-staging.yaml

```diff
@@ -44,6 +44,17 @@ jobs:
           install-kubectl: yes
           install-helm: yes
 
+      - name: Install OpenVPN
+        run: |
+          sudo apt update
+          sudo apt install -y openvpn openvpn-systemd-resolved
+
+      - name: Connect to VPN
+        uses: "kota65535/github-openvpn-connect-action@v2"
+        with:
+          config_file: .github/workflows/config/staging.ovpn
+          client_key: ${{ secrets.STAGING_OVPN_CLIENT_KEY }}     
```

Comment:
Could we have this flow from our 1pw stuff? just wondering here, no blocker

Reply:
hmm that's possible - I like that! Let me look into it!

Reply:
Yup - that works! Thanks for the idea!